### PR TITLE
fix(veeam-backup): fix unmocking

### PR DESCRIPTION
**Pull request by chipp972**
ref: MANAGER-15488

| Question         | Answer
| ---------------- | ---
| Branch?          | `master` 
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | MANAGER-15488 
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

### CODE REVIEW

File: packages/manager/apps/veeam-backup/mocks/iam.mock.ts [hunk 1]
```diff
@@ -1,15 +1,21 @@
...
-const findResourceByUrn = (params: PathParams) =>
-  resourceList.find(({ resourceURN }) => resourceURN === params.urn);
-
-export const getIamMocks = (): Handler[] => [
+export const iamResponse: IamCheckResponse[] = [
+  {
+    urn: backupList[0].iam.urn,
+    authorizedActions: [
+      'account:apiovh:iam/resource/edit',
+      'account:apiovh:service/terminate',
```
Comment (rjamet-ovh) [inline packages/manager/apps/veeam-backup/mocks/iam.mock.ts:9]: you can use your `iamActions` constant on  `packages/manager/apps/veeam-backup/src/veeam-backup.config.ts`

File: packages/manager/apps/veeam-backup/src/data/hook/useOrganizations.ts [hunk 2]
```diff
@@ -1,27 +1,45 @@
...
+    backup?.id.split(backupSuffix)[0]
+  }`;
 
 export const getOrganizationDisplayName = (organization?: VCDOrganization) =>
   organization?.currentState?.fullName || organization?.currentState?.name;
 
-export const getBackupIdFromOrganizationId = (organizationId: string) =>
-  `${organizationId}-veeam-backup`;
+export const getBackupIdFromOrganizationId = (organization: VCDOrganization) =>
+  `${getOrganizationUuid(organization)}${backupSuffix}`;
```
Comment (rjamet-ovh) [inline packages/manager/apps/veeam-backup/src/data/hook/useOrganizations.ts:37]: would be great to test theses functions

File: packages/manager/apps/veeam-backup/src/data/hook/useVeeamBackup.ts [hunk 3]
```diff
@@ -4,6 +4,8 @@ import { ApiError, ApiResponse } from '@ovh-ux/manager-core-api';
 import { VeeamBackupWithIam } from '../vcd.type';
 import { getVmwareCloudDirectorBackup } from '../api';
 
+export const getRegionNameFromAzName = (azName = '') => azName?.split('-a')[0];
+
```
Comment (rjamet-ovh) [inline packages/manager/apps/veeam-backup/src/data/hook/useVeeamBackup.ts:8]: could be tested also

File: packages/manager/apps/veeam-backup/public/translations/order-veeam/Messages_fr_FR.json [hunk 4]
```diff
@@ -11,7 +11,8 @@
   "all_organization_backed_up_message_fetch_next_page": "Toutes les organisations VCD sont déjà associées au service Managed VEEAM for VCD. Veuillez charger plus d'organisations.",
   "all_organization_backed_up_message": "Toutes vos organisations VCD sont déjà associées au service Managed VEEAM for VCD. Rendez-vous sur la liste de vos services.",
   "name_cell": "Nom",
-  "region_cell": "Localisation",
+  "region_cell": "Région",
+  "location_cell": "Localisation",
```
[the remark this answers is not in the document]
Comment (chipp972) [reply packages/manager/apps/veeam-backup/public/translations/order-veeam/Messages_fr_FR.json:15]: It's not a list of veeam-backup here, it's a list of VCD Organization. I'd rather split them.

File: packages/manager/apps/veeam-backup/mocks/iam.mock.ts (hunk 1 above)
Comment (chipp972) [reply packages/manager/apps/veeam-backup/mocks/iam.mock.ts:9]: Yes bien vu

File: packages/manager/apps/veeam-backup/src/data/hook/useOrganizations.ts (hunk 2 above)
Comment (chipp972) [reply packages/manager/apps/veeam-backup/src/data/hook/useOrganizations.ts:37]: yes done

File: packages/manager/apps/veeam-backup/src/data/hook/useVeeamBackup.ts (hunk 3 above)
Comment (chipp972) [reply packages/manager/apps/veeam-backup/src/data/hook/useVeeamBackup.ts:8]: done fixed

**Review by rjamet-ovh — DISMISSED** [review]

**Review by rjamet-ovh — DISMISSED** [review]

**Review by pauldkn — DISMISSED** [review]

**Pull request merged**